specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ManagedEnvironments.json

```diff
@@ -1423,6 +1423,22 @@
                   "$ref": "#/definitions/Mtls"
                 }
               }
+            },
+            "peerTrafficConfiguration": {
+              "description": "Peer traffic settings for the Managed Environment",
+              "type": "object",
+              "properties": {
+                "encryption": {
+                  "description": "Peer traffic encryption settings for the Managed Environment",
+                  "type": "object",
+                  "properties": {
+                    "enabled": {
```

Comment:
>{ [](http://example.com/codeflow?start=31&length=1)

Cosnider making this an enum. 

please also take a look at the other swagger lint warnings here : [https://github.com/Azure/azure-rest-api-specs/pull/28129/checks?check_run_id=22379826783](https://github.com/Azure/azure-rest-api-specs/pull/28129/checks?check_run_id=22379826783)

Reply:
Thanks for your feedback. It's intended to keep consistent with the existing peerAuthentication.mtls.enabled which is also boolean. For the rest "AvoidNestedProperties" warnings, it isn't applicable here.